src/bika/cement/setuphandlers.py

```diff
@@ -43,6 +43,9 @@ def add_dexterity_setup_items(portal):
         ("materialtype_folder",
          "Material Types",
          "MaterialTypeFolder"),
+        ("materialclass_folder",
+         "Material Class",
```

Comment:
Thanks for the PR! Could you please change "Material Class" to "Material Classes".

![image](https://github.com/user-attachments/assets/011e6adc-76b1-45b9-9b96-df3a6bf10fd4)

It would be nice to have the breadcrumbs be the same as the title.

![image](https://github.com/user-attachments/assets/023c77d2-9f70-4288-a91b-514d2ff75403)
![image](https://github.com/user-attachments/assets/c9583a4a-54c5-4c4f-8b95-0caf07359b27)

Reply:
@DanE417